modules/op-release-notes-1-10.adoc

```diff
@@ -0,0 +1,175 @@
+* With this update, for a cluster resolver, bundle resolver, and Git resolver, a pipeline sets the `ConfigSource` value to the source from where the remote resources were received. The `ConfigSource` parameter includes a source URL and hex-encoded sha256 checksum of the cluster resource.
+* This update adds a new boolean flag `enable-provenance-in-status` into the `feature-flags` config map to enable the provenance field in status to be populated. The provenance field records authenticated metadata about how a software artifact was built.
+* This update adds reconciler, event, config, and webhook support for a custom run.
+* This update populates the `Status.Provenance.ConfigSource` parameter for a `TaskRun` or `PipelineRun` by using the value from the remote `ResolutionRequest` status.
++
+[NOTE]
+====
+You need to set the feature flag `enable-provenance-in-status` to `true` to enable the `Status.Provenance.ConfigSource` parameter to be populated and available in a run status.
+====
+
+* With this update, custom tasks in pipelines are enabled by default. 
++
+[NOTE]
+====
+To disable this update, set the `enable-custom-tasks` flag to `false` in the `feature-flags` config map file.
+====
+
+* This update adds support for the `PipelineRun` reconciler to create a custom run. For example, custom `TaskRuns` created from `PipelineRuns` can now use API version `v1beta1.CustomRuns` instead of `v1alpha1.Runs`, if the `custom-task-version` feature flag is set to `v1beta1`, instead of the default value `v1alpha1`. 
++
+[NOTE]
+====
+You need to update the custom task controller to listen for the `*v1beta1.CustomRun` API version instead of `*v1alpha1.Run` in order to respond to `v1beta1.CustomRun` requests.
+====
+
+* This update adds a new parameter `Retries` into `v1beta1.TaskRun` and `v1.TaskRun` API versions.
+* This update includes the following changes in the trusted resources verification package: 
+** Enable Key Management Service (KMS) in the `VerificationPolicy` library for trusted resources.
+** Add a new parameter `KMS` into `v1alpha1.VerificationPolicy` API version. Use this parameter to configure the KMS path and resolve the stored keys.
+** Add KMS libraries to use KMS for verification purposes.
+* This update adds support for tracing using Jaeger and OpenTelemetry. To enable this update, add the following environment variables to the controller manifest:
+** `OTEL_EXPORTER_JAEGER_ENDPOINT`: Denotes the HTTP endpoint for sending spans directly to a collector.
+** `OTEL_EXPORTER_JAEGER_USER`: Denotes the user name to be sent for authentication to the collector endpoint. This is an optional variable.
+** `OTEL_EXPORTER_JAEGER_PASSWORD`: Denotes the password to be sent for authentication to the collector endpoint. This is an optional variable.
```

Comment:
Those are not exposed through the operator though (so not really doable).
cc @jkandasa

Reply:
@vdemeester Do i need to remove this entire bullet point `This update adds support for tracing using Jaeger and OpenTelemetry. To enable this update, add the following environment variables to the controller manifest:`?